content/blog/2021/23/camelDebuggerForIntelliJ/index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Camel Debugger For IntelliJ"
+date: 2021-12-23T07:53:49-07:00
```

Comment:
optional: for the date, it is possible to keep it simpler, and provide just `date: 2021-12-24`